include/ignition/gazebo/EntityComponentManager.hh

```diff
@@ -402,7 +402,7 @@ namespace ignition
       /// a true value should be returned.
       /// \tparam ComponentTypeTs All the desired component types.
       /// \warning This function should not be called outside of System's
-      /// PreUpdate, callback. The result of call after PreUpdate is invalid
+      /// PostUpdate, callback. The result of call after PreUpdate is invalid
```

Comment:
Should the last sentence be removed / updated?

Also, the comma before `callback` seems out of place :monocle_face:

Reply:
> Should the last sentence be removed / updated?

Yes - I forgot about that sentence, thanks for catching it! I have removed it in [78ae5ef](https://github.com/ignitionrobotics/ign-gazebo/pull/348/commits/78ae5efb2a43a382edb1eb7521597a2d6c63ad96).

> Also, the comma before callback seems out of place

Agreed, I have removed this as well in [78ae5ef](https://github.com/ignitionrobotics/ign-gazebo/pull/348/commits/78ae5efb2a43a382edb1eb7521597a2d6c63ad96).